src/main/resources/wa-task-initiation-ia-asylum.dmn

```diff
@@ -105,6 +105,15 @@
             null</text>
         </inputExpression>
       </input>
+      <input id="InputClause_0930dyw" label="Fee update Tribunal Action" camunda:inputVariable="feeUpdateTribunalActions">
+        <inputExpression id="LiteralExpression_00c4nsi" typeRef="string">
+          <text>if(additionalData != null and additionalData.Data != null and additionalData.Data.feeUpdateTribunalActions
+            != null) then
+            additionalData.Data.feeUpdateTribunalActions
```

Comment:
This field should be feeUpdateTribunalAction and not feeUpdateTribunalAction**s**